straxen/plugins/nveto_recorder.py

```diff
@@ -56,6 +56,11 @@ class nVETORecorder(strax.Plugin):
 
     data_kind = {key: key for key in provides}
 
+    def setup(self):
+        
```

Comment:
@WenzDaniel , would you like to have a line 
```python
if isinstance(self.config['nbaseline_samples_lone_records_nv']), int):
  self.baseline_samples = self.config['nbaseline_samples_lone_records_nv']
else:
  self.baseline_samples = straxen.get_correction_from_cmt(self.run_id,
                                                           self.config['nbaseline_samples_lone_records_nv'])
```

Reply:
This solution would work too and is maybe a bit more flexible. Then you also do not have to add a new setup to muVETOProcessing.

Reply:
I would go with the option of adding a `setup()` function for consistency, if you want to change please go ahead and implement it.